app/src/androidTest/java/org/mozilla/tv/firefox/ui/NetworkPageLoadTest.kt

```diff
@@ -31,15 +31,22 @@ class NetworkPageLoadTest {
         val googleUrl = "https://google.com".toUri()!!
         val mozillaUrl = "https://mozilla.org".toUri()!!
 
-        navigationOverlay { }.enterUrlAndEnterToBrowser(googleUrl) {
+        navigationOverlay {
+            // Work around for #1444: on emulators with turbo mode enabled, the url bar will display
+            // any url loaded by the page in addition to the primary url. We disable turbo mode to
+            // ensure we only see the primary url and can assert it correctly.
+            toggleTurbo() // enabled by default.
```

Comment:
Optional nit: it might be good to break this into `turnTurboOn` and `turnTurboOff`, each with an assertion, similarly to `BrowserRobot#openOverlay`.

Reply:
Filed https://github.com/mozilla-mobile/firefox-tv/issues/1464